docs/graphql/core/deployment/deployment-guides/digital-ocean-one-click.rst

```diff
@@ -81,9 +84,8 @@ Navigate to ``Data -> Create table`` on the console and create a table called ``
 
 Choose ``id`` as the Primary key and click the ``Create`` button.
 
-.. image:: https://graphql-engine-cdn.hasura.io/heroku-repo/assets/hasura_create_table.png
-   :class: no-shadow
-   :alt: Hasura console - create table
+.. thumbnail:: /img/graphql/core/deployment/digital-ocean-create-table.png
```

Comment:
@codeliezel how did you compress the images? The quality is quite low. Did you maybe compress them several times?

![image](https://user-images.githubusercontent.com/5722022/97147527-eff8d880-1769-11eb-9fe6-e9a17464cf53.png)

Reply:
I have rectified this